CMakeLists.txt

```
@@ -94,7 +108,7 @@ else()
 endif()
 
 # ----- pika
-find_package(pika 0.4.0 REQUIRED)
+find_package(pika 0.5.0 REQUIRED)
```

To do before merging:
```suggestion
find_package(pika 0.6.0 REQUIRED)
```